nni/retiarii/hub/pytorch/autoformer.py

```diff
@@ -356,3 +420,5 @@ def forward(self, x):
         x = self.head(x)
 
         return x
+
```

Comment:
Did you use a formatter that automatically added this? If not, please delete those.

If you're to use one, we recommend autopep8 with line-length set to 140.